src/cci/cci_common.c

```diff
@@ -80,6 +67,12 @@
 static const float CCI_MHT_REHASH_TRESHOLD = 0.7f;
 static const float CCI_MHT_REHASH_FACTOR = 1.3f;
 
+#if defined(USE_HASHVAL_ORDERED_LIST)
```

Comment:
these two macros seem to disable code blocks. what is the difference?
```
#if defined(USE_HASHVAL_ORDERED_LIST)
#if defined(ENABLE_MEANINGLESS_CODE)
```

Reply:
ENABLE_MEANINGLESS_CODE
  >>  Irrespective of this fix, remove meaningless code because it is not used in existing code.

USE_HASHVAL_ORDERED_LIST
   >>  Allows to maintain sorted order in the newly added hash value comparison processing (not currently defined)
USE_HLS_ORDERED_LIST
   >> Same purpose as USE_HASHVAL_ORDERED_LIST, except that the target is mht_hls.

As a result, ENABLE_MEANINGLESS_CODE is for disabling unused parts of the existing code,
The other two are for future use through further review.

Reply:
@ hyunikn  
You may want to put the actual deletion on hold for now, and then rearrange it later.